src/WinRT.Runtime2/InteropServices/LanguageExceptionErrorInfoMethods2.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Runtime.InteropServices;
+using WindowsRuntime.InteropServices.Marshalling;
+
+namespace WindowsRuntime.InteropServices;
+
+internal static class ILanguageExceptionErrorInfo2Methods
+{
+    public static unsafe void CapturePropagationContext(void* thisPtr, Exception ex)
```

Comment:
Nits, for consistency:
- Rename the exception parameter `exception` (never use abbreviations)
- The reference value should be named the param name + "Value", so here it'd be "exceptionValue"